galgebra/mv.py

```diff
@@ -1911,7 +1916,7 @@ def correlation(u, v, dec=3):  # Compute the correlation coefficient of vectors
     return ulocal.dot(vlocal) / (ulocal.norm() * vlocal.norm()). evalf(dec)
 
 
-def cross(v1, v2):
+def cross(v1: Mv, v2: Mv) -> Mv:
```

Comment:
Why sometimes `'Mv'` and sometimes `Mv`?

Reply:
Within `class Mv`, `Mv` doesn't exist yet, so can't be used directly.

Reply:
Oh.......

Reply:
Note they all render the same way in the docs: https://galgebra--305.org.readthedocs.build/en/305/generated/galgebra.mv.html#module-galgebra.mv

Reply:
Not necessarily: https://stackoverflow.com/a/33533514

Reply:
But as long as we support Python below 3.7, using string is inevitable.

Reply:
Indeed, but we still support 3.5 and 3.6, so this is the preferred approach for now.